code/PostProcessing/GenVertexNormalsProcess.cpp

```diff
@@ -158,9 +158,9 @@ bool GenVertexNormalsProcess::GenMeshVertexNormals (aiMesh* pMesh, unsigned int
 
     // Set up a SpatialSort to quickly find all vertices close to a given position
     // check whether we can reuse the SpatialSort of a previous step.
-    SpatialSort* vertexFinder = NULL;
+    SpatialSort* vertexFinder = nullptr;
     SpatialSort  _vertexFinder;
-    ai_real posEpsilon = ai_real( 1e-5 );
```

Comment:
We may use the max epsilon, but not initializing it is not a good idea ( was a finding from a static code analysis tool )

Reply:
I have also some tool for this small things and that value is not used below.